client/tabular.js

```diff
@@ -24,6 +24,7 @@ var tabularOnRendered = function () {
   template.tabular.searchFields = null;
   template.tabular.searchCaseInsensitive = true;
   template.tabular.tableName = new ReactiveVar(null);
+  template.tabular.throttleRefresh = new ReactiveVar(null);
```

Comment:
Since throttleRefresh is only used on the server, there is no need for any of the changes in this file.